lib/config/folderconfiguration.go

```diff
@@ -25,7 +25,7 @@ import (
 var (
 	ErrPathNotDirectory = errors.New("folder path not a directory")
 	ErrPathMissing      = errors.New("folder path missing")
-	ErrMarkerMissing    = errors.New("folder marker missing (this indicates potential data loss, search docs/forum to get information about how to proceed)")
+	ErrMarkerMissing    = errors.New("folder marker missing (consult the 'Error' section on this page: https://docs.syncthing.net/intro/gui.html?highlight=.stfolder )")
```

Comment:
```suggestion
	ErrMarkerMissing    = errors.New("folder marker missing (consult the 'Error' section on this page: https://docs.syncthing.net/intro/gui.html)")
```

Adding a link to the Docs is useful, but please don't remove the original message though. What if the Docs website isn't working when the problem happens? The user will be left with nothing.

Reply:
I think the highlighting is deliberate - at least it does point to the right place.